[2025-03-15T23:16:58.171+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-15T23:16:58.219+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_wether_dag.api scheduled__2025-03-15T16:11:00+00:00 [queued]>
[2025-03-15T23:16:58.250+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_wether_dag.api scheduled__2025-03-15T16:11:00+00:00 [queued]>
[2025-03-15T23:16:58.252+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-15T23:16:58.302+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): api> on 2025-03-15 16:11:00+00:00
[2025-03-15T23:16:58.344+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2980) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-15T23:16:58.352+0000] {standard_task_runner.py:72} INFO - Started process 2984 to run task
[2025-03-15T23:16:58.359+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_wether_dag', 'api', 'scheduled__2025-03-15T16:11:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/dag_wether.py', '--cfg-path', '/tmp/tmpv_t_d0bv']
[2025-03-15T23:16:58.376+0000] {standard_task_runner.py:105} INFO - Job 34: Subtask api
[2025-03-15T23:16:58.494+0000] {task_command.py:467} INFO - Running <TaskInstance: api_wether_dag.api scheduled__2025-03-15T16:11:00+00:00 [running]> on host 157b2d7c5f40
[2025-03-15T23:16:59.395+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_wether_dag' AIRFLOW_CTX_TASK_ID='api' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T16:11:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-15T16:11:00+00:00'
[2025-03-15T23:16:59.399+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-15T23:16:59.806+0000] {python.py:240} INFO - Done. Returned value was: ok
[2025-03-15T23:16:59.872+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-15T23:16:59.874+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_wether_dag, task_id=api, run_id=scheduled__2025-03-15T16:11:00+00:00, execution_date=20250315T161100, start_date=20250315T231658, end_date=20250315T231659
[2025-03-15T23:16:59.999+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-15T23:17:00.038+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-15T23:17:00.043+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
